src/almond/enginemanagerclient_k8s.ts

```diff
@@ -0,0 +1,219 @@
+// -*- mode: typescript; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Almond
+//
+// Copyright 2021 The Board of Trustees of the Leland Stanford Junior University
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import * as events from 'events';
+import * as rpc from 'transparent-rpc';
+import express from 'express';
+import * as k8s from '@kubernetes/client-node';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const wsjs = require('ws');
+import WebSocket from 'ws';
```

Comment:
`createWebSocketStream` is attached as a static method to the WebSocket class, so a cleaner thing here is to let TypeScript now about it:
```typescript
declare module 'ws' {
    import * as Stream from 'stream';
    interface WebSocket {
        createWebSocketStream(ws : WebSocket, options : { ... }) : Stream.Duplex;
    }
}
```

A `declare module` statement in the middle of a normal file (not in a .d.ts file on its own) augments an existing declaration file. See the [handbook](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation)

Reply:
In fact, it's already declared at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ws/index.d.ts#L348 ?